src/main/java/com/internet/shop/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.UserDao;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.models.ShoppingCart;
+import com.internet.shop.models.User;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Inject
+    private UserDao userDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public User create(User user) {
+        userDao.create(user);
+        shoppingCartService.create(new ShoppingCart(user.getId()));
```

Comment:
should all users have a ShoppingCart? and admin too?
https://mate-academy.github.io/jv-program-fulltime/02_jdbc_and_servlets/checklist/hw_14_2.html
last point